projects/plugins/jetpack/_inc/client/recommendations/feature-utils.js

```diff
@@ -151,6 +158,16 @@ export const mapDispatchToProps = ( dispatch, featureSlug ) => {
 
 export const getStepContent = stepSlug => {
 	switch ( stepSlug ) {
+		case 'backup-plan':
+			return {
+				question: __( 'Be prepared for auto-updates.', 'jetpack' ),
+				description: __(
```

Comment:
This description section as a single block of text isn't very readable imo, The google doc has it sectioned out like this: 

![image](https://user-images.githubusercontent.com/65001528/184447339-978fd4eb-a20e-44eb-8ddf-d11d2cf25220.png)

and personally I think that would be easier to read. In [this PR](https://github.com/Automattic/jetpack/pull/25391) I enabled <br /> tags in the prompts to make this possible 😄

Reply:
Ahh very nice. I'll likely use the same implementation here. We can rebase whichever PR is merged later to sort the conflict.